### PR TITLE
#11372 Added HAVING functions

### PR DESCRIPTION
**Pull request by michanismus**

### CODE REVIEW

File: phalcon/mvc/model/query/builderinterface.zep [hunk 1]
```diff
@@ -196,13 +196,65 @@ interface BuilderInterface
...
-	public function having(having);
+    /**
+     * Sets the HAVING conditions
+     *
+     * @param string conditions
+     * @param array bindParams
+     * @param array bindTypes
+     * @return \Phalcon\Mvc\Model\Query\BuilderInterface
+     */
+    public function having(conditions, bindParams = null, bindTypes = null) -> <BuilderInterface>;
```
Comment (duythien) [inline phalcon/mvc/model/query/builderinterface.zep]: Styles code should be to 4 space

Comment (michanismus) [inline phalcon/mvc/model/query/builderinterface.zep]: Strange thing is, I used 4 spaces...

**Comment by sergeyklay** [comment]
1. Provide please small test case for each method
2. Update `CHANGELOG.md`
3. We use tabs in `zep` files for indents

**Comment by michanismus** [comment]
Ok, i will add tests, update CHANGELOG.md and change spaces to tabs

**Comment by michanismus** [comment]
Ok, added fixed files with tabs for indents, a test for each function and I updated CHANGELOG.md.

**Comment by sergeyklay** [comment]
Could you please return back files mode  100644 → 100755.
Also rebase please onto 2.1.x

**Pull request closed without merging**